lapce-ui/src/plugin.rs

```diff
@@ -158,6 +158,24 @@ impl Widget<LapceTabData> for Plugin {
             let viewport = ctx.size().to_rect().inflate(-padding, 0.0);
             ctx.clip(viewport);
 
+            if data.plugins.is_empty() {
+                let y = self.line_height;
+                let x = self.line_height;
+                let layout = ctx
+                    .text()
+                    .new_text_layout("Failed to load plugin information")
+                    .font(
+                        data.config.ui.font_family(),
+                        data.config.ui.font_size() as f64,
+                    )
+                    .default_attribute(TextAttribute::Weight(FontWeight::SEMI_BOLD))
+                    .text_color(Color::from_hex_str("#d29200").unwrap())
```

Comment:
This should use a color from the theme. Likely `"lapce.warn"` or `"lapce.error"` would work.